src/components/footer/EsFooter.vue

```diff
@@ -138,20 +146,33 @@
 
     watch: {
       $route(to) {
+        const getServiceTypeId = to.fullPath.split('nails/')[1];
+        const getGiftsId = to.fullPath.split('carduri-cadou/')[1];
         this.isProPage = to.fullPath.includes('easepro');
         this.isProWave = to.fullPath.includes('easepro/');
-        this.isClientWave = to.fullPath.includes('client/') || to.fullPath.includes('pro/');
+        this.isClientWave = to.fullPath.includes('client/') || 
+          to.fullPath.includes('pro/') || 
+          to.fullPath.includes('/servicii/masaj') || 
+          to.fullPath.includes(`servicii/nails/${getServiceTypeId}`) || 
+          to.fullPath.includes('abonamente/massage?tip=monthly') || 
+          to.fullPath.includes('abonamente/rezerva');
         this.isContactWave = to.fullPath.includes('contact');
         this.isFAQWave = to.fullPath.includes('intrebari-frecvente');
+        this.isServices = to.fullPath.includes('servicii/masaj/') || to.fullPath.includes('servicii/couple/');
```

Comment:
same here. Define all the links. Create multiple arrays. One for services, one for client wave. it will make the code clean

Reply:
Re-factored.